code/game/objects/items/devices/scanners.dm

```diff
@@ -114,7 +114,8 @@ MASS SPECTROMETER
 
 // Used by the PDA medical scanner too
 /proc/healthscan(mob/living/user, mob/living/M, mode = 1)
-
+	if(user.stat || user.eye_blind)
+		return

```

Comment:
I had fixed this bug once already but somebody reverted it somehow. You shouldn't see the analyzer's results if unconscious or blind.